lua/entities/gmod_wire_teleporter.lua

```diff
@@ -169,7 +169,12 @@ function ENT:Jump_Part2( withangles )
 	self.LocalPos = {}
 	self.LocalAng = {}
 	self.LocalVel = {}
-	for _, ent in pairs( self.Entities ) do
+	for k, ent in ipairs( self.Entities ) do
+        if not IsValid(ent) then
+            table.RemoveByValue(self.Entities, ent)
```

Comment:
Why remove by value when you can remove the object at `k`? Regardless this isn't safe because modifying array indices in ipairs will cause skips.

Reply:
I'm doing remove by value because it keeps the table properly indexed, i see the issue you mean though yeah

Reply:
Oh, so it's just setting that index to `nil`? Neither are very good solutions.

Reply:
RemoveByValue uses table.remove which would keep the indexes properly set, i've changed it up to just use pairs + tbl[k] = nil though.